### ISSUE TITLE
Find musicbrainz id's for the following datasets

### ISSUE THREAD

**Issue by mvanniekerk** [mvanniekerk/musicbrainz_search#32]
- http://www.dtic.upf.edu/~ocelma/MusicRecommendationDataset/index.html
- http://www.cp.jku.at/datasets/MMTD/

**Comment by informusica** [comment]
btw, forgot to mention this morning: listening history also seems to be entering the MusicBrainz/AcousticBrainz initiative: https://listenbrainz.org/
I heard about it from some colleagues last week...still seems rather in beta, but this could also be interesting to explore later on (also by seeing to what extent sparsity is an issue here for classical listeners).

As for the large Music Listening Histories dataset: http://ddmal.music.mcgill.ca/research/musiclisteninghistoriesdataset . I think it will be better to start looking at it at a later stage, but just wanted to put the URL already here.

**Comment by mvanniekerk** [comment]
Ok, this is giving me some ideas: 
- the [last fm 1k](http://www.dtic.upf.edu/%7Eocelma/MusicRecommendationDataset/lastfm-1K.html) dataset has 17 mil tracks (960 000 unique) that have a musicbrainz id
- and 2 mil tracks (450 000 unique) that do not have an id
- It is possible to correlate the track id unambiguously with a work id
- I could see how well the system performs given a work id and a track title for those 960 000 test cases.
- If it performs well, I could find try to find id's for the tracks that are not labeled. This could potentially be useful for large scale labeling (for example in the listenbrainz project).

**Comment by informusica** [comment]
Exactly. I would expect work- vs. track-level resolution to make considerable difference for the classical genre. At the same time, for popular genres, I think it should not make that much difference (as the concept of 'track' vs. 'work' may be nearly identical in those genres, so it should not be harmful).
